pages/index.py

```diff
@@ -61,6 +61,37 @@ def slider_item(title, text, active=""):
 <!--HOME SECTION START-->
 <section id="home">
 <div class="container">
+<div class="row">
+<!--DONATE SUBSECTION -->
+<div class="col-sm-12 col-md-12 col-lg-12">
+        <div class="item active donate" id="donate_button">
+             <p>$donate_text</p>
```

Comment:
Good catch, though note this PR didn't introduce this issue. Should I fix it in this PR, or submit a separate one?